pom.xml

```diff
@@ -217,6 +217,12 @@
         <version>${spark.version}</version>
         <scope>${spark.deps.scope}</scope>
       </dependency>
+      <dependency>
```

Comment:
Why is it needed for whole project scope? I think it will only be used in streaming related modules

Reply:
ok